tests/unit/test_arg_helpers.py

```diff
@@ -150,10 +149,3 @@ def test_remove_plugin_not_available(self, mocked_config):
         msg, code = arg_helpers.remove_plugin("testing.plugin", mocked_config)
         assert "not a registered plugin" in msg
         assert code == 14
-
-    def test_bake_command_bad_website(self, capsys, mock_cli):
```

Comment:
I decided to remove this error handler/test because the detailed error gives more valuable information (especially when it comes to cert issues):

```
Failed to load spec: HTTPSConnectionPool(host='fake.linode.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x10f38a5d0>: Failed to resolve 'fake.linode.com' ([Errno 8] nodename nor servname provided, or not known)"))
```